common/src/main/java/io/netty/util/internal/OneTimeTask.java

```diff
@@ -15,14 +15,11 @@
  */
 package io.netty.util.internal;
 
-import io.netty.util.concurrent.EventExecutor;
 
 /**
- * {@link Runnable} which represent a one time task which may allow the {@link EventExecutor} to reduce the amount of
- * produced garbage when queue it for execution.
- *
- * <strong>It is important this will not be reused. After submitted it is not allowed to get submitted again!</strong>
+ * @deprecated Use {@link Runnable}.
  */
+@Deprecated
 public abstract class OneTimeTask extends MpscLinkedQueueNode<Runnable> implements Runnable {
```

Comment:
@Scottmitch well Im not super strong here but I think it also is not really hurting to keep it for one or two more releases... There is no "overhead" in just keep it. @trustin WDYT ?